baselines/Train.py

```diff
@@ -72,7 +89,10 @@ def test(epoch, testloader, model, criterion, args, device):
             for i in range(len(dates)):
                 date = dates[i]
                 try:
-                    efi_tensor = torch.as_tensor(ds_efi.sel(time=date).to_numpy()).to(device)
+                    if (args.model == 'Tformer') or (args.model == 'LeNet'):
+                        efi_tensor = torch.reshape(torch.as_tensor(ds_efi.sel(time=date).to_numpy()).to(device), (361,720))
```

Comment:
why reshape?

Reply:
I reshape the `efi_tensor` for tformer since when it's loaded longitude and latitude are stacked (`259920` instead of `(361,720)`). And the tformer intialy outputs a map `(361,720)` rather than sequence of grid points compared to EMOS and MLP. The LeNet output is just reshaped at the end of the forward pass to match the tformer implementation. I guess we can also reshape the output for both tformer and lenet to match `(259920)` so we don't have to reshape the `efi_tensor` here. should I change it this way?

Reply:
Yes, that would looks better.

Reply:
I will change that too then.

Reply:
"Problem" now is that I need to change some of the `loader.py` code, since the means, stddevs and targets were also reshape to the grid `(361, 720)`, will update here when everything works.